pkg/jsonc/jsonc_test.go

```diff
@@ -26,3 +26,13 @@ func TestToJSON(t *testing.T) {
 		t.Fatalf("expected '%s', got '%s'", expect, out)
 	}
 }
+
+func TestToJSON2(t *testing.T) {
```

Comment:
@aleh-null do we still need this test?

Reply:
Probably a good idea to have test on the actual dev container. WDYT of using https://github.com/marcozac/go-jsonc in the future?

Reply:
is it better?